src/main/java/tool/injury/InjuryDetailPanel.java

```diff
@@ -110,17 +109,20 @@ final int getTSIPre() {
         }
     }
 
+
+    private static final DBManager.PreparedStatementBuilder preStatementBuilder = new DBManager.PreparedStatementBuilder(
+            "select marktwert from SPIELER where spielerid=? and verletzt=-1 order by DATUM desc");
+    private static final DBManager.PreparedStatementBuilder postStatementBuilder = new DBManager.PreparedStatementBuilder(
+            "select marktwert from SPIELER where spielerid=? and verletzt>-1 order by DATUM desc");
```

Comment:
We should push those SQL queries out of the view.  Not sure where the best place is, but grouping those SQL queries in a controller-type class would be preferable.  Maybe in `Player`?

Reply:
changed to: 
~~~
           String tsi = player.getLatestTSINotInjured();
...
            tsiPre.setText(tsi);

            tsi = player.getLatestTSIInjured();
~~~

Reply:
Definitely much better, thanks!